src/MyStateless/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Do we really need this file to commit?

Reply:
Hmm probably not, no idea where it comes from :-)

Reply:
Found it, when you debug the solution from within visual studio it will open the browser and navigate to the demo controller after the application is deployed to the cluster so you do not have to perform this step manually.